mri/numerics/reconstruct.py

```diff
@@ -96,6 +99,16 @@ def sparse_rec_fista(gradient_op, linear_op, prox_op, cost_op,
         print(" - alpha variable shape: ", alpha.shape)
         print("-" * 40)
 
+    # Define the proximity dual operator
+    weights = copy.deepcopy(alpha)
+    weights[...] = mu
+    prox_op.weights = weights
+
+    beta_param = gradient_op.inv_spec_rad
+    if lambda_update_params.get("restart_strategy") == "greedy":
+        lambda_update_params["min_beta"] = gradient_op.inv_spec_rad
+        beta_param *= 1.3
```

Comment:
Where does this value come from, could you comment please?

Reply:
sure, tell me if it's enough

Reply:
Perfect, Thanks